### PR TITLE
PLAT-687: Validate VFindCallRequest fields

### PR DESCRIPTION
**Pull request by dgovorukhin**
Add validation method for VFindCallRequest. Checking fields value invariants.

### CODE REVIEW

**Comment by bigbes** [comment]
No checks of RequestedContent/RequestedContentLimit (unused for now)/Supported extensions (unused for now)

**Comment by bigbes** [comment]
Don't mind my comment, missed message

File: ledger-core/insolar/payload/validation.go [hunk 1]
```diff
@@ -177,3 +179,44 @@ func (m *VStateReport) validateUnimplemented() error {
 
 	return nil
 }
+
+func (m *VFindCallRequest) Validate(currPulse PulseNumber) error {
+	var (
+		lookAtPulse   = m.GetLookAt()
+		calleePulse   = m.GetCallee().GetLocal().Pulse()
```
Comment (bigbes) [inline ledger-core/insolar/payload/validation.go:186]: please, add check that callee is valid SelfRef (`IsSelfScope`), for now. That check will be replaced with `IsObjectReference` in the future

Comment (dgovorukhin) [reply ledger-core/insolar/payload/validation.go:186]: sure, not problem)

Comment (dgovorukhin) [reply ledger-core/insolar/payload/validation.go:186]: Fixed

File: ledger-core/insolar/payload/validation.go [hunk 2]
```diff
@@ -177,3 +179,44 @@ func (m *VStateReport) validateUnimplemented() error {
...
+		outgoingPulse = m.GetOutgoing().GetLocal().Pulse()
+	)
+
+	switch {
+	case lookAtPulse < pulse.LocalRelative:
```
Comment (bigbes) [inline ledger-core/insolar/payload/validation.go:195]: I've found out that there's helper function on PulseNumber: `IsValidAsPulseNumber`

Comment (dgovorukhin) [reply ledger-core/insolar/payload/validation.go]: Done

**Pull request merged**